packages/sirius-web/frontend/sirius-web/src/onboarding/NewDocumentArea.types.ts

```diff
@@ -10,8 +10,11 @@
  * Contributors:
  *     Obeo - initial API and implementation
  *******************************************************************************/
-import { Selection } from 'workbench/Workbench.types';
+import { Selection } from '../../../../../core/frontend/sirius-components-core/src/workbench/Workbench.types';
```

Comment:
This would not work, use `@eclipse-sirius/sirius-components-core`. The same comment can be applied to other locations.